scripts/helpers/tmux_utils.sh

```diff
@@ -1,3 +1,6 @@
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
This breaks update plugins functionality because in `scripts/update_plugin_prompt_handler.sh` this line redefines `CURRENT_DIR` which is used for calling `"$CURRENT_DIR/update_plugin.sh" --tmux-echo "$*"`

Reply:
I think we don't need `CURRENT_DIR` in this helper script so I removed it.